VMEncryption/main/oscrypto/OSEncryptionStateMachine.py

```diff
@@ -31,7 +31,7 @@
 transitionsdir = os.path.abspath(os.path.join(scriptdir, '../../transitions'))
 sys.path.append(transitionsdir)
 
-from OSEncryptionState import *
+from .OSEncryptionState import *
```

Comment:
does this work on python2?  if not new syntax may be needed

Reply:
explicit relative imports work the same on python2 (they just aren't required like python3)